src/oc_erchef/apps/chef_db/src/chef_sql.erl

```diff
@@ -176,6 +178,16 @@ is_user_in_org(UserName, OrgName) ->
             {error, Error}
     end.
 
+-spec get_server_admins_authz_id() -> boolean() | {error, _}.
+get_server_admins_authz_id() ->
```

Comment:
Let's keep this in `oc_chef_authz_db` since it's also used within an `oc_chef_authz` module - we already may have a circular dependency there, and the less we add to it, the easier it will be to untangle.